packages/platform-sdk-profiles/src/environment/services/exchange-rate-service.ts

```diff
@@ -7,8 +7,18 @@ import { ProfileRepository } from "../../repositories/profile-repository";
 import { ReadWriteWallet, WalletData } from "../../wallets/wallet.models";
 import { container } from "../container";
 import { Identifiers } from "../container.models";
+import { DateTime } from "@arkecosystem/platform-sdk-intl";
 
 export class ExchangeRateService {
+	readonly #lastSynced: Record<string, DateTime> = {};
+	readonly #cacheDurationMilliseconds: number = 10000;
```

Comment:
Treat the ttl in seconds. We are never caching it based on milliseconds so we should stick to seconds or minutes for the ttl.